src/parser/hunter/beastmastery/modules/spells/CobraShot.tsx

```diff
@@ -138,7 +138,8 @@ class CobraShot extends Analyzer {
       >
         <BoringSpellValueText spell={SPELLS.COBRA_SHOT}>
           <>
-            {formatNumber(this.effectiveKCReductionMs / 1000)}s / {this.totalPossibleCDR / 1000}s <br />
+            {formatNumber(this.effectiveKCReductionMs / 1000)}s / {this.totalPossibleCDR / 1000}s
```

Comment:
Same dealio. Probably should check if bad divides no matter how rare 
As they say at my work users data is dirty